arm/Microsoft.Network/virtualNetworks/.parameters/min.parameters.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "sxx-az-vnet-min-001"
```

Comment:
I'd call it `sxx-az-vnet-x-002` for consistency with other modules

Reply:
I thought the `x` is there to allow us to give it a a dedicated name :D Let's discuss naming next week :)

Reply:
Using parameter-`prefix` for `x`